src/Lib/Managers/ModeManager.vala

```diff
@@ -0,0 +1,181 @@
+/**
+ * Copyright (c) 2021 Alecaddd (http://alecaddd.com)
```

Comment:
nit: https :P

Reply:
Lol, fixed. Pretty sure I copy/pasted this.